CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### Bug fixes
+
+* [#184](https://github.com/rubocop-hq/rubocop-rails/issues/184): Fix Rake/Environment to allow task with no block. ([@hanachin][])
```

Comment:
Can you enclose the cop name in backticks?

```diff
-* [#184](https://github.com/rubocop-hq/rubocop-rails/issues/184): Fix Rake/Environment to allow task with no block. ([@hanachin][])
+* [#184](https://github.com/rubocop-hq/rubocop-rails/issues/184): Fix `Rake/Environment` to allow task with no block. ([@hanachin][])
```

Reply:
Yes I can :muscle: